week-5/F-webchat/exercise-1.js

```diff
@@ -33,5 +34,11 @@ When you open index.html in your browser, it should display the existing message
 
 */
 
+// Write your code here
 
-// Write your code here
\ No newline at end of file
+fetch("https://codeyourfuture.herokuapp.com/api/messages")
+  .then(response => response.text())
+  .then(function(mes) {
+    document.querySelector("#message-list").innerHTML = mes;
```

Comment:
You should use messages.forEach() to append each message separately as a new node

Reply:
mes.forEach(element => {
  element[content];
});
U MEAN  LIKE THIS,,,,,

Reply:
```suggestion
messages.forEach(message => {
            var paragraph = document.createElement("p");
            paragraph.appendChild(document.createTextNode(message.datetime + ": " + message.content));
            messageList.appendChild(paragraph);
        });
```